app/models/thredded/topic.rb

```diff
@@ -15,6 +15,23 @@ class Topic < ActiveRecord::Base
 
     scope :order_sticky_first, -> { order(sticky: :desc) }
 
+    scope :followed_by, lambda { |user|
+      joins('INNER JOIN thredded_user_topic_follows ON thredded_user_topic_follows.topic_id = thredded_topics.id')
+        .where('thredded_user_topic_follows.user_id = ?', user.id)
```

Comment:
There are several ways to do this without using sql strings:

1. `where(thredded_user_topic_follows: {user_id: user.id})`
2. `merge(Thredded::UserTopicFollow.where(user_id: user.id))`
3. `merge(user.thredded_topic_follows)`